apps/provisioning_api/lib/Controller/UsersController.php

```diff
@@ -300,6 +300,96 @@ public function getDisabledUsersDetails(?int $limit = null, int $offset = 0): Da
 		]);
 	}
 
+	/**
+	 * @NoAdminRequired
+	 * @NoCSRFRequired
+	 *
+	 * Get the list of disabled users and their details
+	 *
+	 * @param string $search Text to search for
+	 * @param ?int $limit Limit the amount of users returned
+	 * @param int $offset Offset
+	 * @param string $sortMode Field to order the results with
+	 * @param string $sortOrder asc or desc
+	 * @return DataResponse<Http::STATUS_OK, array{users: array<string, Provisioning_APIUserDetails|array{id: string}>}, array{}>
+	 *
+	 * 200: Users details returned based on last logged in information
+	 */
+	public function getLastLoggedInUsers(string $search = '',
+										 ?int   $limit = null,
+										 int    $offset = 0,
+										 string $sortMode = 'lastLogin',
+										 string $sortOrder = 'desc'
```

Comment:
```suggestion
	public function getLastLoggedInUsers(string $search = '',
										 ?int   $limit = null,
										 int    $offset = 0,
										 string $sortOrder = 'desc'
```
Let's keep it simple as we only need sorting by last login

Reply:
users/recent will work as it is without any params (default sort is by lastLogin desc) and in case asc/desc is required frontend can pass users/recent?offset=0&limit=25&search=&sortOrder=asc&sortBy=lastLogin params accordingly.